src/utils/envVars.ts

```diff
@@ -1,10 +1,12 @@
 export const IS_MAINNET                 = Boolean(process.env.REACT_APP_IS_MAINNET !== 'false');  // If REACT_APP_IS_MAINNET is unset, set it to true by default
+export const TESTNET_LAUNCHPAD_NAME      = 'Pyrmont'
 
 // private vars (or derived from)
 export const PORTIS_DAPP_ID             = process.env.REACT_APP_PORTIS_DAPP_ID     || '';
 export const INFURA_PROJECT_ID          = process.env.REACT_APP_INFURA_PROJECT_ID  || '';
 export const ENABLE_RPC_FEATURES        = Boolean(INFURA_PROJECT_ID && INFURA_PROJECT_ID !== '');
 export const INFURA_URL                 = `https://${IS_MAINNET ? "mainnet" : "goerli"}.infura.io/v3/${INFURA_PROJECT_ID}`;
+export const BEACONCHAIN_ENDPOINT       = `https://${IS_MAINNET ? `` : `${TESTNET_LAUNCHPAD_NAME.toLowerCase()}.`}beaconcha.in/api/v1/epoch/latest`;
```

Comment:
We already have a `BEACONCHAIN_URL` variable in this file. Let's just use that.

e.g. so in queryBeaconchain.ts:
```
axios.get(`${BEACONCHAIN_URL}/api/v1/epoch/latest)
```

Reply:
Good call, fixing now.